lib/polo/sql_translator.rb

```diff
@@ -7,47 +9,35 @@ class SqlTranslator
     def initialize(object, configuration=Configuration.new)
       @record = object
       @configuration = configuration
-    end
-
-    def to_sql
-      records = Array.wrap(@record)
 
-      sqls = records.map do |record|
-        raw_sql(record)
-      end
-
-      if @configuration.on_duplicate_strategy == :ignore
-        sqls = ignore_transform(sqls)
+      case @configuration.adapter
+      when :mysql then self.extend(Polo::Adapters::MySQL)
```

Comment:
Not sure I like this extend pattern much.
We could simplify things and pass in the adapter as an object in the constructor.
e.g. `def initialize(object, configuration=Configuration.new, adapter)`

That would simplify our tests a lot too.